### ISSUE TITLE
Bad CPU error on Mac for standalone cli 

### ISSUE THREAD

**Issue by sorenpeter** [marp-team/marp-cli#620]
- OS version: macOS Montery 12.7.6 (MacBook Pro Retina, 15-inch, Mid 2015)

When trying to use the standalone marp cli I get this error:

```bash
 $ ./marp slides.md 
-bash: ./marp: Bad CPU type in executable
```

Are the standalone only made or M1+ and not Intel Macs?

**Comment by yhatt** [comment]
Yes, the standalone binary for Mac is built for Apple Silicon. You can still install Marp CLI to Intel Mac via [Homebrew](https://github.com/marp-team/marp-cli#macos-homebrew) or [npm](https://github.com/marp-team/marp-cli#local-installation).

**Comment by sorenpeter** [comment]
Hi yhatt,

Thanks for the clarification. I got Marp CLI working via homebrew (but it took close to an hour to install...)

Any change you will label the binary and make a note in the readme that they are only for Silicon, to avoid the confusion for others in the future?
